documentation/howto/index.adoc

```diff
@@ -13,6 +13,8 @@ link:mobiledev_createwebapp.html[Create Your First Mobile Web App]
 
 link:hmt_firstapp.html[Create Your First Hybrid Mobile App]
 
+link:import_FH_app.adoc[Import Your FeedHenry App]
```

Comment:
this should be import_FH_app.html not adoc.

Also, could you please use lowercase naming for adoc and the images as done for all the other content we got under docs ? thanks.